### PR TITLE
feature: add networkpolicy and azure spot toleration

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: helm-charts/falcon-sensor/templates/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{{- if .Values.container.enabled -}}
+{{- if .Values.container.networkPolicy.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "falcon-sensor.fullname" . }}-default-deny-ingress
+  namespace: falcon-system
```
Comment (domdepasquale) [inline helm-charts/falcon-sensor/templates/networkpolicy.yaml:7]: {{ .Release.Namespace }}

File: helm-charts/falcon-sensor/templates/networkpolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "falcon-sensor.fullname" . }}-network-policy
+  namespace: falcon-system
```
Comment (domdepasquale) [inline helm-charts/falcon-sensor/templates/networkpolicy.yaml:25]: {{ .Release.Namespace }}

File: helm-charts/falcon-sensor/templates/networkpolicy.yaml (hunk 1 above)
Comment (redhatrises) [reply helm-charts/falcon-sensor/templates/networkpolicy.yaml]: Thanks! Totally missed that. Fixed.

File: helm-charts/falcon-sensor/templates/networkpolicy.yaml (hunk 2 above)
Comment (redhatrises) [reply helm-charts/falcon-sensor/templates/networkpolicy.yaml]: Thanks! Totally missed that. Fixed.

**Pull request merged**